src/test/java/unit/com/mathworks/ci/actions/RunMatlabBuildActionTest.java

```diff
@@ -130,40 +107,4 @@ public void shouldPrintAndRethrowMessage() throws IOException, InterruptedExcept
             assertEquals(12, e.getExitCode());
         };
     }
-
-    @Test
-    public void shouldNotAddActionIfNoBuildResult() throws IOException, InterruptedException, MatlabExecutionException {
-        action.run();
-
-        verify(build, never()).addAction(any(BuildArtifactAction.class));
-    }
-
-    @Test
-    public void shouldCopyBuildResultsToRootAndAddAction() throws IOException, InterruptedException, MatlabExecutionException {
-        File tmp = Files.createTempDirectory("temp").toFile();
-        tmp.deleteOnExit();
-        
-        File dest = Files.createTempDirectory("dest").toFile();
-        dest.deleteOnExit();
-
-        File json = new File(tmp, "buildArtifact.json");
-        json.createNewFile();
-
-        doReturn(new FilePath(tmp)).when(runner).getTempFolder();
-        doReturn(dest).when(build).getRootDir();
-
-        action.run();
-
-        // Should have deleted original file
-        assertFalse(json.exists());
-        // Should have copied file to root dir
-        assertTrue(new File(dest, "buildArtifact"+ action.getActionID() + ".json").exists());
-    }
-
-    @Test
-    public void shouldRemoveTempFolder() throws IOException, InterruptedException, MatlabExecutionException {
-        action.run();
-
-        verify(runner).removeTempFolder();
-    }
```

Comment:
I don't think that this should be removed.

Reply:
Good catch, I unintentionally removed this test. Moving it to `RunMatlabActionTest.java`